src/integrationTest/java/uk/gov/hmcts/reform/sendletter/e2e/BaseTest.java

```diff
@@ -105,32 +106,43 @@ private boolean letterHasBeenPosted() {
         return letters.size() == 1 && letters.get(0).getStatus() == LetterStatus.Posted;
     }
 
-    private void createXeroxReport(LocalSftpServer server, Boolean isEncryptionEnabled) throws IOException {
-        try (Stream<UUID> letterIds = Arrays.stream(server.lettersFolder.listFiles())
-            .map(file -> validateGettingLetterId(file, isEncryptionEnabled))) {
+    private void createXeroxReport(LocalSftpServer server) throws IOException {
+        try (Stream<UUID> letterIds = Arrays.stream(server.lettersFolder.list())
+            .map(FileNameHelper::extractIdFromPdfName)) {
```

Comment:
we upload .zip (or .pgp), not .pdf, and file name format for pdf and zip are different. what am I missing?

Reply:
In both cases the letter id is the penultimate component of the name